rosdep/python.yaml

```diff
@@ -1513,6 +1513,22 @@ python-fastdtw-pip:
   ubuntu:
     pip:
       packages: [python-fastdtw]
+python-fastnumbers-pip:
```

Comment:
Just to be clear; this is implicitly a package for Python 2.  Are you using it for Python 2?  If so, this is OK as-is.  If not, I'd prefer to make this:

```
python3-fastnumbers-pip
```

Reply:
Good point, I oversaw that completely. I was using it with Python 2.7 and it seems to work just fine, but I double checked the documentation on PyPi because I was not completely sure anymore whether 2.7 is actually supported and turns out it's for versions >= 3.5.
I'll update it ASAP !